react/week2/react-w2/src/App.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import './App.css';
+
+
+const todos = [
+    {
+        id: 1,
+        description: "Get out of bed",
+    },
+    {
+        id: 2,
+        description: "Brush teeth",
+    },
+    {
+        id: 3,
+        description: "Eat breakfast",
+    },
+];
+
+const myTodoList = ["Finish my hyf homework", "Walk the dog", "Do laundry", "Prepare for class"]
+
+function CreateTodo(props) {
+    const [todoStatus, setTodoStatus] = React.useState(true);
+    function checkState() {
+        setTodoStatus(!todoStatus)
+    }
+
+    return (
+        <div>
+          <ul>
+        <li style={{ 'textDecoration': todoStatus ? 'none' : 'line-through' }} >{props.description}</li>
+                <input type="checkbox" onClick={checkState}></input>
+                <button onClick={() => props.deleteMethod(props.id)}> Delete </button>
+           </ul>
+        </div>
+    )
+}
+
+function TodoListContainer() {
+    return (
+        <div>
+            <h1>Todo List</h1>
+            <TodoList />
+        </div>
+    )
+}
```

Comment:
In general I think your idea is good, but in this case I think it is a little unnecessary to abstract just the header out to a single component. I would personally just have the header inside the TodoList component.

Reply:
Will do that from next time